### PR TITLE
update footer.html and footer.scss

### PR DESCRIPTION
**Pull request by Viktoriia181818**

### CODE REVIEW

File: src/partials/footer.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
+<footer class="footer">
+    <div class="section">
+        <div class="container">
+            <h2 class="footer-title">CONTACT US</h2>
+            <div class="section-footer">
+                <div class="footer-address">
```
Comment (Rekovich) [inline src/partials/footer.html]: footer__address

File: src/partials/footer.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                    <p class="footer-title-text">Get in touch with us</p>
```
Comment (Rekovich) [inline src/partials/footer.html]: footer__title

File: src/partials/footer.html [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                    <p class="footer-text">
```
Comment (Rekovich) [inline src/partials/footer.html]: footer__text

File: src/partials/footer.html [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+                        Nunc urna est, tristique et varius at, pharetra nec
+                        enim. Quisque rhoncus lacus fringilla nulla sollicitudin
+                        tristique.
+                    </p>
+
+                    <ul class="footer-socials">
+                        <li class="footer-socials-net">
```
Comment (Rekovich) [inline src/partials/footer.html]: footer-socials__item

File: src/partials/footer.html [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                            <a class="team-socials__social--networks" target="_blank" rel="noopener noreferrer"
```
Comment (Rekovich) [inline src/partials/footer.html]: footer-socials__link

File: src/partials/footer.html [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+                                href="#">
+                                <svg class="team-socials__icon" width="40" height="40">
```
Comment (Rekovich) [inline src/partials/footer.html]: footer-socials__icon

File: src/partials/footer.html [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+                                +xx-xxxx-xxxx-xx
+                            </li>
+                            <li class="footer-contact__email">
+                                goIteens@gmail.com
+                            </li>
+                        </ul>
+                    </address>
+                </div>
+                <form class="form">
+                    <div class="contact-form">
```
Comment (Rekovich) [inline src/partials/footer.html]: contact contact__form

File: src/partials/footer.html [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+                        <label class="contact-name" for="name">
```
Comment (Rekovich) [inline src/partials/footer.html]: contact__name

File: src/partials/footer.html [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+                            <input class="contact-name" placeholder="Name" type="text" name="name" id="name">
```
Comment (Rekovich) [inline src/partials/footer.html]: contact__name

File: src/partials/footer.html [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+                        </label>
+                        <label class="contact-name contact-email" for="contact-email">
```
Comment (Rekovich) [inline src/partials/footer.html]: contact__name--email

File: src/partials/footer.html [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+                            <input class="contact-name contact-email" placeholder="Email" type="email"
+                                name="contact-email" id="contact-email">
+                        </label>
+                    </div>
+                    <label class="contact-textarea" for="textarea" id="contact-textarea">
```
Comment (Rekovich) [inline src/partials/footer.html]: contact__textarea

File: src/partials/footer.html [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+                        <textarea class="textarea" placeholder="Message" name="contact-textarea" id="textarea" cols="30"
+                            rows="10"></textarea>
+                    </label>
+                    <div class="button-class">
+                        <button type="button" class="footer-button">
```
Comment (Rekovich) [inline src/partials/footer.html]: button footer__button

**Review by EugeneNikiforov — APPROVED** [review]

**Review by Rekovich — APPROVED** [review]

**Pull request merged**